lib/stateView/contract/updateSVContractFactory.js

```diff
@@ -13,32 +13,30 @@ function updateSVContractFactory(svContractRepository) {
    * @param {string} userId
    * @param {Reference} reference
    * @param {Contract} contract
-   * @param {boolean} [reverting]
+   * @param {MongoDBTransaction} [transaction]
    *
    * @returns {Promise<void>}
    */
-  async function updateSVContract(contractId, userId, reference, contract, reverting = false) {
+  async function updateSVContract(
```

Comment:
Aren't we agreed that we're not going to allow contract updates, at least for the beginning?

Reply:
Yes, it's implemented in DPP.

Reply:
@antouhou @shumkov ?

Reply:
We don't need to do anything here. This logic is already in `dpp.packet.verify` (`verifySTPacket`) function.